### DISCUSSION TITLE
possible to have target in the chart header

### DISCUSSION THREAD

**Discussion by gavinsboyle** [sly9/VoyagerTelegramBot#35] [Ideas]
hi guys

currently the title of the stats chart includes my sequence name - which for me running a structured (kind of "perpetual")dragscript is typically "automated plan 1".
is there a way we could have an option to include the target name in that header too - or the ability to choose between the sequence name and the target name as the addendum to the generic chart title?

thanks
gavin

**Comment by sly9** [comment]
I think this is actually very related / similar to one of the things we plan to do for mosaic — one sequence file with multiple ‘targets’(Panes), and right now we can’t differentiate that

I’ll create an issue from this thread and track it
